src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1291,7 +1291,7 @@ internal enum ErrorCode
         INF_UnableToLoadSomeTypesInAnalyzer = 8040,
         // available 8041-8049
         ERR_InitializerOnNonAutoProperty = 8050,
-        ERR_AutoPropertyMustHaveGetAccessor = 8051,
+        // ERR_AutoPropertyMustHaveGetAccessor = 8051, Removed with semi auto properties.
```

Comment:
>ERR_AutoPropertyMustHaveGetAccessor [](http://example.com/codeflow?start=11&length=35)

If we don't need the error, we should also remove the corresponding message.

Reply:
@AlekseyTs I'll need to get the error back. See https://github.com/dotnet/roslyn/pull/57076#discussion_r773327444